### PR TITLE
Use ramsey/composer-install

### PR DESCRIPTION
**Pull request by greg0ire**
It makes for shorter workflow files.

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -75,31 +78,22 @@ jobs:
...
+      - name: "Enforce using stable dependencies"
+        run: "composer config minimum-stability stable"
+        if: "${{ matrix.stability == 'stable' }}"
 
-      - name: "Install dev dependencies with composer"
-        run: "composer update --no-interaction --prefer-dist"
-        if: "${{ matrix.deps == 'dev' }}"
-
-      - name: "Install lowest possible dependencies with composer"
-        run: "composer update --no-interaction --prefer-dist --prefer-stable --prefer-lowest"
```
Comment (ostrolucky) [inline .github/workflows/continuous-integration.yml:101]: --prefer-lowest job needs to exist

**Review by ostrolucky — CHANGES_REQUESTED** [review]
CI is failing

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (greg0ire) [reply .github/workflows/continuous-integration.yml:101]: It does exist :)

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -25,30 +25,36 @@ jobs:
...
+        stability:
+          - "stable"
         symfony-require:
           - ""
         symfony-deprecations-helper:
           - ""
         include:
           # Tests the lowest set of dependencies
-          - deps: "low"
+          - dependencies: "lowest"
```
Comment (greg0ire) [inline .github/workflows/continuous-integration.yml:38]: @ostrolucky here is why it exists

**Review by ostrolucky — APPROVED** [review]

**Pull request merged**